src/unittest/test_voxelarea.cpp

```diff
@@ -120,7 +120,7 @@ void TestVoxelArea::test_extent()
 	VoxelArea v1(v3s16(-1337, -547, -789), v3s16(-147, 447, 669));
 	UASSERT(v1.getExtent() == v3s16(1191, 995, 1459));
 
-	VoxelArea v2(v3s16(32493, -32507, 32753), v3s16(32508, -32492, 32768));
+	VoxelArea v2(v3s16(32493, -32507, 32753), v3s16(32508, -32492, -32768));
```

Comment:
If you do this you're actually testing for an overflow, I suggest instead subtracting 1 from every component to keep it in-range.

Reply:
```suggestion
	VoxelArea v2(v3s16(32493, -32507, 32752), v3s16(32508, -32492, 32767));
```
likewise below

Reply:
This was already checking overflow behavior before, so I left it so as to not change behavior, but that was probably not intended.  I'll make that change.